src/pybind/mgr/dashboard_v2/controllers/perf_counters.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from ..tools import ApiController, AuthRequired, RESTController
+
+
+class BasePerfCounter(RESTController):
+    _service_type = None
+
+    @property
+    def service_type(self):
+        return self._service_type
```

Comment:
Why add this property? Just use `service_type`: 
```
class BasePerfCounter(RESTController):
    service_type = None
```

Reply:
I always use properties to ensure that it is used read-only and i want to define a clear interface that the user can use. This is a way of mine to simulate protected/private class members in Python.